### PR TITLE
rtmp-services: Add Garena platform

### PR DESCRIPTION
**Pull request by businiaowyf**
This PR aims at adding Garena Live as a streaming platform. Garena Live is one of the largest streaming platform in South East Asia. This is our official site: [https://garena.live/](https://garena.live/). Thank you very much!

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -1068,6 +1068,28 @@
                 "max video bitrate": 6000,
                 "max audio bitrate": 160
             }
+        },
+        {
+            "name": "Garena Live",
+            "common": true,
```
Comment (jumoog) [inline plugins/rtmp-services/data/services.json]: Garena.tv is no common Service please remove it

Comment (businiaowyf) [reply plugins/rtmp-services/data/services.json]: OK, I already removed it, thank you

**Comment by notr1ch** [comment]
@businiaowyf I have contacted gca@garena.com regarding GPL infringements in the GarenaTV software (it's based on OBS Studio). If you're able to push this along internally it would be appreciated.

**Comment by businiaowyf** [comment]
@Richard, could you tell me more details about the infringement? I will do
my best to push it and give feedback to you soon as possible.

**Comment by notr1ch** [comment]
@businiaowyf GarenaTV appears to be using a lot of code from OBS Studio, but I saw no offer of source code or mention of the GPL during the download or installation. I searched github and Google and didn't find the source code for GarenaTV available anywhere. As OBS Studio is licensed under the GPL, it's required that other projects using the code are also licensed under the GPL, and that requires the source code for GarenaTV to be available.

**Pull request closed without merging**